src/OpenLoco/src/Windows/News/Ticker.cpp

```diff
@@ -225,9 +212,8 @@ namespace OpenLoco::Ui::Windows::NewsWindow::Ticker
         {
             _word_525CE0 = _word_525CE0 | (1 << 15);
         }
-        uint32_t ebp = (((_word_525CE0 & ~(1 << 15)) >> 2) << 16) | 109;
 
-        sub_4950EF(&*clipped, StringIds::buffer_2039, (1 << 18), ebp, 55, 0);
+        drawingCtx.drawStringTicker(*clipped, { 55, 0 }, StringIds::buffer_2039, Colour::black, 4, ((_word_525CE0 & ~(1 << 15)) >> 2), 109);
```

Comment:
I would rather have the logic from this function moved in a static function in this file.